adapters/smarthub/smarthub.go

```diff
@@ -0,0 +1,187 @@
+package smarthub
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"text/template"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/macros"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+const (
+	ADAPTER_VER = "1.0.0"
+)
+
+type SmartHubAdapter struct {
```

Comment:
Please rename this to `adapter`. There's no need to repeat the package name in the adapter type. Please review our new adapter docs: https://docs.prebid.org/prebid-server/developers/add-new-bidder-go.html

Reply:
Thank you, I've been fixed this. Do other things look correct?